src/app/store/account.slice.ts

```diff
@@ -133,3 +172,21 @@ export function* removeAccountWorker({ payload: account }: PayloadAction<IAccoun
     });
   }
 }
+
+export function* generateAccountWorker() {
+  const mnemonicPhrase: string = yield call(ipcBridgeRenderer.crypto.invoke, {
```

Comment:
Could anything go wrong here? Do we need error handling?

Reply:
In theory any IPC call can go wrong, but I don't think there's a big need to handle errors here.